### DISCUSSION TITLE
Trigger hotkey only when target is hovered

### DISCUSSION THREAD

**Discussion by alexisthual** [palantir/blueprint#5745] [Q&A] [answered]
#### Environment

- __Package version(s)__: 4.0.0-beta.12
- __Browser and OS versions__: irrelevant

#### Feature request

Setting `global: false` for a given hotkey allows to enable event only when target is focused.
I would like to like to have the same behaviour available for hovering (ie hotkey works if and only if target is hovered).

#### Examples

I am building an interface with a lot of panes. All panes have the same hotkey combos, but these combos trigger different callbacks depending on the pane.
Navigation would be much more fluid if users didn't have to click on a pane for it to become active and then use hotkeys.

**Comment by adidahiya** [comment, marked as answer]
I think you could implement this behavior inside your hotkey event handler. Track which element is being hovered over in your application state, and react accordingly. There's no great way for the Blueprint hotkeys code to do this out of the box in the same way that global/local hotkeys are supported with element focus & the native event bubbling system.
